scripts/generator/miscGenerator.ts

```diff
@@ -79,6 +79,7 @@ const miscAddresses: Addresses = {
     value: '0xc85b1E333aecc99340b2320493Fe2d22b8734795',
   },
   AAVE_SWAPPER_ETHEREUM: '0x3ea64b1C0194524b48F9118462C8E9cd61a243c7',
+  AAVE_POL_ETH_BRIDGE: '0xc928002904Ec475663A83063D492EA2aE09EbDA1',
```

Comment:
```suggestion
  AAVE_POL_ETH_BRIDGE: {
      chainId: ChainId.polygon,
      type: 'address',
      value: '0xc928002904Ec475663A83063D492EA2aE09EbDA1',
  },
```
should probably be like this (if no network is specified we assume mainnet/etherscan)